mobile/src/main/res/values/strings.xml

```diff
@@ -242,6 +247,13 @@
     <string name="error_sending_alarm_clock_item_empty">Please enter a valid Item name</string>
     <string name="retry">Retry</string>
     <string name="ignore">Ignore</string>
+    <string name="send_device_info_schedule">Schedule</string>
+    <string name="send_device_info_schedule_summary">These Items are updated every 15 minutes while charging, every %s otherwise.\nNote: Lower values impair battery life more than higher ones. Your device may decide to delay updates.</string>
```

Comment:
May also want to move the `Note:` part to a separate explanation preference, to make the fact the first part refers to the set value stand out a bit more.

Reply:
I'd first try to make the `%s` bold and/or italic, so it stands out more.

Reply:
This one is still open ... for consistency, I think I'd prefer the separate info pref.

Reply:
![Screenshot_2020-04-23-21-31-29](https://user-images.githubusercontent.com/22525368/80141583-0d29d000-85aa-11ea-9eb8-143a7f72f136.png)
![Screenshot_2020-04-23-21-32-19](https://user-images.githubusercontent.com/22525368/80141584-0dc26680-85aa-11ea-9f36-1d700669de5d.png)

What version do you prefer? I prefer the second one as it's more consistent with the event based category.

Reply:
Yes, second version is much better.

Reply:
About consistency: one hint has a period, the other one doesn't ;-)

Reply:
Done